db/controller/article-controller.js

```diff
@@ -0,0 +1,24 @@
+const {
+    selectArticles,
+    selectAllArticles
+} = require('../model/articles-model');
+
+const getAllArticles = (request, response, next) => {
+    selectAllArticles()
+    .then((articles) => {
+        response.status(200).send({articles: articles})
+    })
+    .catch(next);
+}
+
+const getArticles = (request, response, next) => {
+    const article_id = parseInt(request.params.article_id);
```

Comment:
Parsing the id to a number could cause problems in error handling when making sure it's the right data type the user has passed